README.md

```diff
@@ -22,7 +22,9 @@ Once you've updated the `ruby-build` plugin then all you need to do is run: `rbe
 3. run `bundle install --binstubs .bundle/bin`
 4. run `rbenv rehash`
 
-
+## Configure environment variables
+1. Install the [rbenv-vars plugin](https://github.com/sstephenson/rbenv-vars).
+2. In the project root folder create a `.rbenv-vars` and add the necessary environment variables.
```

Comment:
It's a bit vague. It would be nice if you could specify what those actually are.
This is basically the same as my argument against `options`.

Anyway, not really important for this project.